common/static/common/js/components/views/list.js

```diff
@@ -24,18 +24,24 @@
                 this.itemViews = [];
             },
 
+            renderCollection: function() {
+                // Render every item in the collection.
```

Comment:
Nit: I'd make this a JSDoc comment for the function, rather than an inline comment.